packages/app-admin-ui/client/pages/Home/components.js

```diff
@@ -76,7 +75,7 @@ export const Name = styled.span(
   font-size: 1.1em;
   font-weight: 500;
   max-width: 100%;
-  overflow: hidden;
+  overflow-x: hidden;
```

Comment:
```suggestion
  overflow: hidden;
```

Reply:
`overflow-y` is what caused the letter bottoms to be cut off. I stuck with `overflow-x` so long labels wouldn't extend past the right edge of the card.

Reply:
It was really the line-height that was causing the issue not the overflow.

Reply:
Ah, you're right! Done.